hw_16_conv_store.py

```diff
@@ -0,0 +1,132 @@
+class Product:
+    def __init__(self, name: str, price: int | float, unit: int | float):
+        self.name = name
+        self.price = price
+        self.unit = unit
+
+    def __repr__(self):
+        return self.name
+
+    def __float__(self):
+        return float(self.price)
```

Comment:
Коли ціна вказана у `int` мають на увазі копійки.
```suggestion
        return self.price / 100
```